08-Secure by default/Secure Templates/Bicep/Storage.bicep

```diff
@@ -15,6 +15,7 @@ resource storageaccount 'Microsoft.Storage/storageAccounts@2021-04-01' = {
   }
   kind: 'StorageV2' //[Azure_Storage_AuthN_Dont_Allow_Anonymous]
   properties: {
+    minimumTlsVersion: 'TLS1_2' //Azure_Storage_DP_Use_Secure_TLS_Version_Trial
```

Comment:
Doesn't this need to be enclosed by square brackets like the others?
Also, please confirm by actually using ARM and Bicep template to create a TLS compliant Storage account.

Reply:
Added square bracket. 
Yes it will create TLS compliant Storage by using ARM & Bicep Template